iocore/net/P_SSLConfig.h

```diff
@@ -161,6 +159,33 @@ struct SSLCertificateConfig {
   static int configid;
 };
 
+struct SSLTicketParams : public ConfigInfo {
+  ssl_ticket_key_block *default_global_keyblock;
+  char *ticket_key_filename;
+  void LoadTicket();
+  void cleanup();
+  ~SSLTicketParams();
+};
+
+struct SSLTicketKeyConfig {
+  static void startup();
+  static bool reconfigure();
+  static SSLTicketParams *
+  acquire()
+  {
+    return static_cast<SSLTicketParams *>(configProcessor.get(configid));
+  }
+  static void
```

Comment:
As was pointed out on the mailing list, we really prefer an empty line after the } on these functions. I know clang-format can mess with this, but see if you can get it to behave?